ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -72,7 +76,10 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        update_packages: True
+        copr: 'abbra/wip-ipa-trust'
```

Comment:
This repo contains a sssd build and an IPA build needed for ipa 2 ipa trust , but you can’t install both this pr’s ipa build and the copr ipa build. Let’s use only your PR for now.